packages/@react-native-windows/cli/src/config/configUtils.ts

```diff
@@ -291,6 +291,92 @@ export function importProjectExists(
   return nodes.length > 0;
 }
 
+export type ConfigurationType =
+  | 'application'
+  | 'dynamiclibrary'
+  | 'generic'
```

Comment:
what is 'generic'?

Reply:
_Makefile, displays makefile toolset (NMake)_ according to [ConfigurationTypes documentation](https://docs.microsoft.com/en-us/dotnet/api/microsoft.visualstudio.vcproject.configurationtypes?view=visualstudiosdk-2019).

Reply:
gotcha. We don't support nmake (msbuild support of nmake is limited to drivers, I think?) but I guess it doesn't hurt to leave it here. Feel free to remove if you want.